environments/prod/platform-hmcts-net.yml

```diff
@@ -16,9 +16,9 @@ ns:
   - name: "demo"
     ttl: 900
     record:
-      - "ns1-02.azure-dns.com"
-      - "ns2-02.azure-dns.net"
-      - "ns3-02.azure-dns.org"
+      - "ns1-02.azure-dns.com."
+      - "ns2-02.azure-dns.net."
+      - "ns3-02.azure-dns.org."
```

Comment:
somehow we ended up with the same nameservers, but I've added the . which should've been there anyway